diff -uNdr ssr-0.4.4_old/scripts/ssr-glinject ssr-0.4.4_new/scripts/ssr-glinject
--- ssr-0.4.4_old/scripts/ssr-glinject	2021-05-24 21:56:32.000000000 +0300
+++ ssr-0.4.4_new/scripts/ssr-glinject	2022-04-05 16:31:10.892708483 +0300
@@ -59,6 +59,6 @@
 	fi
 done
 
-echo "ssr-glinject: LD_PRELOAD = $LD_PRELOAD:libssr-glinject.so" >& 2 || true
+echo "ssr-glinject: LD_PRELOAD = $LD_PRELOAD:/usr/\$LIB/simplescreenrecorder/libssr-glinject.so" >& 2 || true
 echo "ssr-glinject: command = $@" >& 2 || true
-LD_PRELOAD="$LD_PRELOAD:libssr-glinject.so" exec "$@"
+LD_PRELOAD="$LD_PRELOAD:/usr/\$LIB/simplescreenrecorder/libssr-glinject.so" exec "$@"
diff -uNdr ssr-0.4.4_old/src/AV/Input/GLInjectInput.cpp ssr-0.4.4_new/src/AV/Input/GLInjectInput.cpp
--- ssr-0.4.4_old/src/AV/Input/GLInjectInput.cpp	2021-05-24 21:56:32.000000000 +0300
+++ ssr-0.4.4_new/src/AV/Input/GLInjectInput.cpp	2022-04-05 16:32:09.524635452 +0300
@@ -207,7 +207,7 @@
 bool GLInjectInput::LaunchApplication(const QString& channel, bool relax_permissions, const QString& command, const QString& working_directory) {
 
 	// prepare command
-	QString full_command = "LD_PRELOAD=\"libssr-glinject.so\" ";
+	QString full_command = "LD_PRELOAD=\"/usr/\\$LIB/simplescreenrecorder/libssr-glinject.so\" ";
 	full_command += "SSR_CHANNEL=\"" + ShellEscape(channel) + "\" ";
 	if(relax_permissions)
 		full_command += "SSR_STREAM_RELAX_PERMISSIONS=1 ";
